src/kvstore/raftex/RaftPart.cpp

```diff
@@ -28,7 +28,7 @@ DEFINE_uint64(raft_snapshot_timeout, 60 * 5, "Max seconds between two snapshot r
 
 DEFINE_uint32(max_batch_size, 256, "The max number of logs in a batch");
 
-DEFINE_int32(wal_ttl, 86400, "Default wal ttl");
+DEFINE_int32(wal_ttl, 14400, "Default wal ttl");
```

Comment:
current is 4 hour ?

Reply:
Current is 1 day, @whitewum suggest change it shorter.